Composer/packages/client/src/telemetry/AppInsightsClient.ts

```diff
@@ -29,14 +29,16 @@ export default class AppInsightsClient {
     }
   }
 
-  public static trackEvent(name: string, properties: LogData) {
-    if (alwaysTrackEvents.includes(name as TelemetryEventName) || this._telemetrySettings?.allowDataCollection) {
+  public static trackEvent(name: TelemetryEventName, properties: LogData) {
```

Comment:
Make sure this builds correctly. I tried this on my last telemetry PR, and there were a ton of type errors. Might be better to just leave it as a string for the moment.

Reply:
This does build - the only messing around with types I had to do was to add TestEvent to the "Other" set, since that's what we use in unit tests. If we don't to make a new type just for testing, we could make the unit tests use some other existing event type.

Reply:
Nah, that's fine.